crispy_bootstrap5/bootstrap5.py

```diff
@@ -7,7 +7,6 @@ class FloatingField(Field):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.attrs["placeholder"] = self.fields[0]
```

Comment:
This change means we can now remove the whole `__init__` function?

Reply:
Fixed